src/amo/components/AddonMeta.js

```diff
@@ -1,37 +0,0 @@
-import React, { PropTypes } from 'react';
```

Comment:
Yeah, hmm. Maybe you can remove everything except for the user count and then it could sit on the left of the install switch.